modules/system/console/OctoberEnv.php

```diff
@@ -184,6 +217,10 @@ private function buildCallback($envKey, $configKey)
 
             $this->saveEnvSettings($envKey, $value);
 
+            if ($this->option('protect') && array_key_exists($envKey, $defaultConfig)) {
```

Comment:
Does this actually work as it is right now? I don't see where you are actually pulling the $defaultConfig variable into this function for using.